src/software/jetson_nano/services/network.cpp

```diff
@@ -113,7 +112,7 @@ void NetworkService::worldCallback(TbotsProto::World input)
     if (packet_loss_rate > PACKET_LOSS_WARNING_THRESHOLD)
     {
         LOG(WARNING) << "World loss in the past "
-                     << std::min(seq_num + 1,
+                     << std::min(seq_num,
                                  static_cast<uint64_t>(RECENT_PACKET_LOSS_PERIOD))
                      << " packets is more than " << PACKET_LOSS_WARNING_THRESHOLD * 100
```

Comment:
nit: instead of saying "more than x%", I think it would be more descriptive if we just give the actual loss rate `packet_loss_rate`. Same for the other log message.

Reply:
Sounds good. I'll keep the threshold, but when the loss rate is greater than the threshold, I'll log the actual loss rate.